exporting/prometheus/multi_metadata.yaml

```diff
@@ -0,0 +1,379 @@
+# yamllint disable rule:line-length
+---
+id: 'export-prometheus-remote'
```

Comment:
I think this needs to be a list starting with "-" with &id.
@Ferroin can you confirm?